style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  box-size: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+@import url(https://fonts.googleapis.com/css?family=roboto);
+
+html {
+  font-family: "roboto", arial;
+  font-size: 18px;
+}
+
+nav {
+  justify-content: center;
+  padding: 20px;
+  background-color: #046c94;
+  color: #fff;
+  list-style-type: none;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-links li {
+  list-style-type: none;
+}
+
```

Comment:
I don't think this is being used anymore since you no longer have li in your nav links